lib/fetch-fares.js

```diff
@@ -112,7 +112,7 @@ const triasJourneyMatchesOtpItinerary = (logger, it) => (j) => {
 		return false
 	}
 	let itDep = +new Date(itLeg0.startTime)
-	if ('number' === itLeg0.departureDelay) itDep -= itLeg0.departureDelay * 1000 // todo: seconds?
+	if ('number' === typeof itLeg0.departureDelay) itDep -= itLeg0.departureDelay * 1000 // todo: seconds?
```

Comment:
Good catch!

If you're changing this anyways: `Number.isInteger()` or `Number.isFinite()` are more precise than using `typeof`.

Also, the "todo: seconds?" comment can be removed I'd say.